georocket-server-api/src/main/java/io/georocket/storage/ChunkMeta.java

```diff
@@ -103,7 +153,8 @@ public JsonObject toJsonObject() {
     return new JsonObject()
         .put("parents", ps)
         .put("start", start)
-        .put("end", end);
+        .put("end", end)
+        .put("$type", ChunkMeta.class.getName());
```

Comment:
`$type` should be added by the caller. The `ChunkMeta` should not need to care about this.